src/Compatibility/Core/src/Android/CollectionView/ItemsSourceFactory.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.Maui.Controls.Compatibility.Platform.Android
 {
 	internal static class ItemsSourceFactory
 	{
-		public static IItemsViewSource Create(IEnumerable itemsSource, ICollectionChangedNotifier notifier)
+		public static IItemsViewSource Create(IEnumerable itemsSource, BindableObject container, ICollectionChangedNotifier notifier)
```

Comment:
@PureWeen I saw this was done for ObservableItemTemplateCollection for Windows, so I copied that. Mainly just for the dispatcher, but if we pass the view in, then we never have to worry about reaching out into the Application. or Device. worlds.

Not sure if this is going to do terrible things with lifetime and/or memory. If so, we can change and find another way.